config/materials/hei-cast_pu-8400.mtl

```diff
@@ -0,0 +1,68 @@
+﻿ND_RelParSet_K01 = {
+
+Name = HEI-CAST_PU-8400
+
+PARAMETERS =
+{
+  Name = PTC_MATERIAL_TYPE
+  Type = Integer
+  Default = 9
+  Access = Locked
+},
+{
+  Name = PTC_FAILURE_CRITERION_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_FATIGUE_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_SUB_TYPE
+  Type = String
+  Default = 'LINEAR'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'Polyurethane resin -vacuum casting urethane for prototypes
+shore 70'
```

Comment:
@Mick3Lozzo the shore hardness for this material can range between 60 and 90. Therefore, we should either remove this property from the description or differentiate the name based on the shore hardness.